### DISCUSSION TITLE
Adding Suricata Rules Airgap

### DISCUSSION THREAD

**Discussion by jasonnin53** [Security-Onion-Solutions/securityonion#4178] [Unsupported Versions]
Good day,

What is the best way to add Suricata rules to Security Onion when it is Airgapped.  I currently do not have any Suricata rules added and would like to add some to Security Onion.  Would it be adding the rules to the /opt/so/saltstack/local/salt/idstools/local.rules for it to populate?  Would we need to restart and update any other things as well?  Also would this be all I need to do to see Suricata alerts in the Alerts tab in Security Onion and for the alerts to show up in Kibana or would I need to do more for it to populate? 

Thank you for your help.

**Comment by TOoSmOotH** [comment]
https://docs.securityonion.net/en/2.3/local-rules.html

If the alert fires, it will show up in elastic which will make it visible for all the things.
